tests/snuba/api/endpoints/test_organization_event_details.py

```diff
@@ -221,3 +221,31 @@ def test_blank_fields(self):
         assert response.status_code == 200, response.content
         assert response.data["id"] == "a" * 32
         assert response.data["projectSlug"] == self.project.slug
+
+    def test_out_of_retention(self):
+        self.store_event(
+            data={
+                "event_id": "d" * 32,
+                "message": "oh no",
+                "timestamp": iso_format(before_now(days=1)),
```

Comment:
Any chance of this being flakey since it's equal to the retention days? If so, would it be good to bump this further back to like 2 days?

Reply:
:+1: good idea